Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -59,6 +59,11 @@ protected static IMixedRealityInputSystem InputSystem
         /// </summary>
         public HashSet<IMixedRealityInputSource> PressingInputSources => pressingInputSources;
 
+        /// <summary>
+        /// Values for user-friendly dimensions settings
+        /// </summary>
+        public enum SelectionTypes { Button, Toggle, MultiDimension };
```

Comment:
Since the new name is SelectionMode, can you update the enum name also?

Reply:
I moved the enum out to it's own file and added clarification on each value.